static/scss/ecommerce.scss

```diff
@@ -59,3 +59,58 @@
     font-size: 16px;
   }
 }
+
+.payment-history {
+  letter-spacing: 1.73px;
+
+  .inner-container {
+    margin: 5px;
+    padding: 20px 15px;
+  }
+
+  .payment-history-header {
+    display: flex;
+    flex-direction: row;
+    align-content: space-between;
+
+    h2 {
```

Comment:
Similar comment to one I left on another PR. This is the payment history header in the design next to a standard `<h1>`. Can we just use `<h1>` for this element and avoid more custom CSS?

<img width="856" alt="ss 2020-06-16 at 14 15 08 " src="https://user-images.githubusercontent.com/14932219/84812147-ef8e4b00-afdb-11ea-904b-713643102119.png">